Formula/bind.rb

```diff
@@ -41,6 +41,9 @@ class Bind < Formula
     sha256 "00c7c1aaa88358b9c765b6d3000c6eec0ba42abca5351b095321aef446081da3"
   end
 
+  # Remove use of -flat_namespace
```

Comment:
We need to upstream this patch if we're using it. The usual patches we use are to fix bugs in Libtool, and that's already been upstreamed.

Reply:
noted